Netwrix/netwrix_auditor/ingest/parser.yml

```
@@ -0,0 +1,87 @@
+name: netwrix_auditor_parser
+pipeline:
+  - name: cef_parser
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general-setter
+    description: ''
+    filter: ''
+    external: null
+  - name: user-email-setter
+    description: ''
+    filter: '{{ "@" in cef_parser.message.suser }}'
+    external: null
+  - name: user-name-domain-setter
+    description: ''
+    filter: '{{ "\\" in cef_parser.message.suser }}'
+    external: null
+  - name: user-only-setter
+    description: ''
+    filter: '{{ "\\" not in cef_parser.message.suser and "@" not in cef_parser.message.suser }}'
+  - name: document-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.cat == "Document" }}'
+    external: null
+  - name: logon-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Logon Activity" }}'
+    external: null
+  - name: exchange-event-setter
+    description: ''
+    filter: '{{ cef_parser.message.DeviceProduct == "Exchange Online" }}'
+    external: null
+stages:
+  general-setter:
+    actions:
+      - set:
+          event.kind: '{{ cef_parser.message.DeviceProduct }}'
+          event.category: '{{ cef_parser.message.DeviceEventClassID |lower }}'
+          event.severity: '{{ cef_parser.message.Severity }}'
+          event.start: '{{ cef_parser.message.start }}'
```

You can also add information about the observer:
```suggestion
          event.start: '{{ cef_parser.message.start }}'
           observer.vendor: '{{parsed_event.message.DeviceVendor}}'
          observer.product: '{{parsed_event.message.DeviceProduct}}'
          observer.version: '{{parsed_event.message.DeviceVersion}}'
```